backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -156,13 +154,25 @@ public Optional<OrganizationRoleClaims> createUser(
             .map(f -> generateFacilityGroupName(organizationExternalId, f.getInternalId()))
             .collect(Collectors.toSet()));
 
-    GroupList orgGroups =
-        _client.listGroups(generateGroupOrgPrefix(organizationExternalId), null, null);
-    if (orgGroups.stream().count() == 0) {
-      throw new IllegalGraphqlArgumentException(
-          String.format(
-              "Cannot add Okta user to nonexistent organization=%s", organizationExternalId));
-    }
+    var orgGroups =
+        Stream.concat(
+                _client
+                    .listGroups(
+                        null,
+                        "profile.name sw \""
+                            + generateGroupOrgPrefix(organizationExternalId)
+                            + "\"",
+                        null)
+                    .stream(),
+                _client
+                    .listGroups(generateGroupOrgPrefix(organizationExternalId), null, null)
+                    .stream())
+            .distinct()
+            .collect(Collectors.toList());
```

Comment:
Combining both `q` and `search` results are necessary because 
1. `q` limits to 200 results
2. `search` results may not have the latest groups if recently added.
By just using the `search` results, when a new organization is created not all of the groups will immediately be available. 

Example funkiness
- q results (orgGroup2/orgGroupNames2)
![image](https://user-images.githubusercontent.com/10108172/184704300-041bdd97-bb98-401d-88be-1759c272e34c.png)
- search results (orgGroup/orgGroupNames)
![image](https://user-images.githubusercontent.com/10108172/184704128-c4bf1b82-f943-4f21-8d0d-cfcdeb252a2a.png)
- search results when `_client.listgroups` was evaluated at a later time
![image](https://user-images.githubusercontent.com/10108172/184704341-f42b723c-d5ff-4697-8283-645db497dbc1.png)

Reply:
nice find!

Reply:
Oh, the Okta API 😭 @BobanL would you mind adding a comment to `LiveOktaRepository` explaining the reason for the duplicate calls? (Referencing the Okta bug you file works there too!)